qemu/tests/cfg/numa_opts.cfg

```diff
@@ -92,3 +93,19 @@
               extra_params += " -numa node,mem=2560,cpus=5,nodeid=2"
               extra_params += " -numa node,mem=512,cpus=3-4,nodeid=0"
               extra_params += " -numa node,mem=1024,cpus=0-2,nodeid=1"
+    - nodes.128:
+        type = numa_maxnodes
+        numa_nodes = 128
+        mem_fixed = 4G
+        use_mem = no
+        node_size = 32M
+        ppc64,ppc64le:
+            mem_fixed = 32G
+            node_size = 256M
+            vm_mem_minimum = 32G
+        paused_after_start_vm = no
+        prealloc_mem = yes
+        policy_mem = default
```

Comment:
I am not familiar with this feature, but in polarion, both x86 and ppc use `policy=bind`,  just want to double confirm with you.

Reply:
@yanan-fu This case is to test 128 nodes, either policy is ok. And if policy=bind, we need the specified host node has all the memory, which could be hard to find such host for ppc.  So we decided to use default policy instead. Thanks.

Reply:
@huangyum  we don't specify `host-nodes=0` as the polarion case, reason is same with above, right ?

Reply:
@yanan-fu Yes, we can not set host node when using default policy.